python/pyspark/ml/wrapper.py

```diff
@@ -223,3 +223,20 @@ def _call_java(self, name, *args):
         sc = SparkContext._active_spark_context
         java_args = [_py2java(sc, arg) for arg in args]
         return _java2py(sc, m(*java_args))
+
+
+class JavaCallable(object):
```

Comment:
JavaCallable seems reasonable.  Could you modify it so that JavaModel can inherit from it and eliminate the duplicate code?